.github/workflows/pr.yaml

```diff
@@ -73,6 +73,8 @@ jobs:
           tool: nextest@0.9.84
       - name: Run Tests
         run: cargo nextest run --all-features --profile ci
+      - name: Run Doc Tests
+        run: cargo test --doc
```

Comment:
I think we might want `--all-features` here in order to test feature docs as well

Reply:
Yeah, makes sense. https://github.com/slatedb/slatedb/pull/469.